### ISSUE TITLE
Issue after latest release

### ISSUE THREAD

**Issue by banesullivan** [awalsh128/cache-apt-pkgs-action#56]
We are seeing an issue after the 1.2 release here: https://github.com/pyvista/pyvista/runs/8214751873?check_suite_focus=true#step:7:123

```
ls: cannot access '/home/runner/cache-apt-pkgs/*.tar': No such file or directory
```

I'm not sure what's going wrong and not sure where to start with debugging this, so I figured I'd make an issue since this release was fairly recent

ref https://github.com/pyvista/pyvista/pull/3273

**Comment by banesullivan** [comment]
Full action log:

<details>

```
Run awalsh128/cache-apt-pkgs-action@v1.2.0
  with:
    packages: libgl1-mesa-glx xvfb python-tk
    version: 1
    refresh: false
  env:
    ALLOW_PLOTTING: true
    SHELLOPTS: errexit:pipefail
    pythonLocation: /opt/hostedtoolcache/Python/3.7.13/x64
    PKG_CONFIG_PATH: /opt/hostedtoolcache/Python/3.7.13/x64/lib/pkgconfig
    Python_ROOT_DIR: /opt/hostedtoolcache/Python/3.7.13/x64
    Python[2](https://github.com/pyvista/pyvista/runs/8214751873?check_suite_focus=true#step:7:2)_ROOT_DIR: /opt/hostedtoolcache/Python/[3](https://github.com/pyvista/pyvista/runs/8214751873?check_suite_focus=true#step:7:3).7.13/x6[4](https://github.com/pyvista/pyvista/runs/8214751873?check_suite_focus=true#step:7:4)
    Python3_ROOT_DIR: /opt/hostedtoolcache/Python/3.7.13/x64
    LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.7.13/x64/lib
Run /home/runner/work/_actions/awalsh128/cache-apt-pkgs-action/v1.2.0/pre_cache_action.sh \
grep: /tmp/cache-apt-pkgs-action-cache/install.log: No such file or directory

19:26:14 Validating action arguments (version='1', packages='libgl1-mesa-glx python-tk xvfb ')...
19:26:14 done

19:26:14 Installing apt-fast for optimized installs...
--2022-09-06 19:26:14--  https://raw.githubusercontent.com/ilikenwf/apt-fast/master/apt-fast
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 18[5](https://github.com/pyvista/pyvista/runs/8214751873?check_suite_focus=true#step:7:5).199.109.133, 185.199.111.133, 185.199.110.133, ...
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.109.133|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 22293 (22K) [text/plain]
Saving to: ‘/usr/local/sbin/apt-fast’

     0K .......... .......... .                               100% 24.3M=0.001s

2022-09-0[6](https://github.com/pyvista/pyvista/runs/8214751873?check_suite_focus=true#step:7:6) 19:26:14 (24.3 MB/s) - ‘/usr/local/sbin/apt-fast’ saved [22293/22293]

19:26:14 done

19:26:14 Verifying packages...
19:26:19 done

19:26:19 Creating cache key...
19:26:19 - Normalized package list is 'libgl1-mesa-glx=21.2.6-0ubuntu0.1~20.04.2 python-tk=2.[7](https://github.com/pyvista/pyvista/runs/8214751873?check_suite_focus=true#step:7:7).1[8](https://github.com/pyvista/pyvista/runs/8214751873?check_suite_focus=true#step:7:8)-1 xvfb=2:1.20.13-1ubuntu1~20.04.3 '.
1[9](https://github.com/pyvista/pyvista/runs/8214751873?check_suite_focus=true#step:7:9):26:19 - Value to hash is 'libgl1-mesa-glx=21.2.6-0ubuntu0.1~20.04.2 python-tk=2.7.18-1 xvfb=2:1.20.13-1ubuntu1~20.04.3  @ 1'.
19:26:19 - Value hashed as 'fed3621cdf09f1a7049be03fb93d3f82'.
19:26:19 done
19:26:19 Hash value written to /home/runner/cache-apt-pkgs/cache_key.md5
Run actions/cache@v3
Received 850898 of 850898 ([10](https://github.com/pyvista/pyvista/runs/8214751873?check_suite_focus=true#step:7:10)0.0%), 8.6 MBs/sec
Cache Size: ~1 MB (850898 B)
/usr/bin/tar --use-compress-program unzstd -xf /home/runner/work/_temp/fcf9b3b0-f31e-4c57-bc[11](https://github.com/pyvista/pyvista/runs/8214751873?check_suite_focus=true#step:7:11)-3ededde[12](https://github.com/pyvista/pyvista/runs/8214751873?check_suite_focus=true#step:7:12)bfe/cache.tzst -P -C /home/runner/work/pyvista/pyvista
Cache restored successfully

Cache restored from key: cache-apt-pkgs_fed3621cdf09f1a7049be03fb93d3f82
Run /home/runner/work/_actions/awalsh128/cache-apt-pkgs-action/v1.2.0/post_cache_action.sh \
grep: /tmp/cache-apt-pkgs-action-cache/install.log: No such file or directory

grep: /tmp/cache-apt-pkgs-action-cache/install.log: No such file or directory

19:26:20 Found 8 files in the cache.
19:26:20 - blt:2.5.3+dfsg-4.tar.gz
19:26:20 - cache_key.md5
19:26:20 - install.log
19:26:20 - libgl1-mesa-glx:21.2.6-0ubuntu0.1~20.04.2.tar.gz
19:26:20 - manifest_all.log
19:26:20 - manifest_main.log
19:26:20 - python-tk:2.7.18-1.tar.gz
19:26:20 - tk8.6-blt2.5:2.5.3+dfsg-4.tar.gz

19:26:20 Reading from main requested packages manifest...
19:26:20 - libgl1-mesa-glx 21.2.6-0ubuntu0.1~20.04.2
19:26:20 - python-tk 2.7.18-1
19:26:20 - xvfb 2 1.20.[13](https://github.com/pyvista/pyvista/runs/8214751873?check_suite_focus=true#step:7:13)-1ubuntu1~20.04.3
19:26:20 done

ls: cannot access '/home/runner/cache-apt-pkgs/*.tar': No such file or directory
Error: Process completed with exit code 2.
```

</details>

**Comment by awalsh128** [comment]
Hi @banesullivan, I think the problem here is that GZip compression was removed so it would cause restores to fail. This is why I bumped the major rev but it should still respect the install. I will look into a migration strategy and patch v1.2.0.

In the meantime, change your version input on the action to something else. This will perform a fresh install and should not be an error on restore. Let me know what your results are.

```yaml
- uses: awalsh128/cache-apt-pkgs-action@latest
  with:
    packages: ...
    version: 1.0 <-- something else
```
